src/workspaces/utils.ts

```diff
@@ -206,6 +206,14 @@ export const containsProtectedDataPolicy = (policies: WorkspacePolicy[] | undefi
     policies
   );
 
+export const containsPhiTrackingPolicy = (policies: WorkspacePolicy[] | undefined): boolean => {
```

Comment:
Does this need to be potentially undefined? In events, there's a check for `'policies' in workspaceObject` before calling this function. Could `hasPhiTrackingPolicy` do the same?

Reply:
The check that is there isn't sufficient. The build error I get if I don't allow `undefined` is below.

I don't particularly feel like it's worth rewriting the calling sites of this method.

```
  Type 'undefined' is not assignable to type 'WorkspacePolicy[]'.

228     'policies' in workspaceObject ? containsProtectedDataPolicy(workspaceObject.policies) : undefined;

Reply:
Fair enough. Not really specific to this PR, but generally it's nice if we can narrow types as high up as possible to avoid lower level functions / components having to deal with multiple cases / missing values.